types/three/three-core.d.ts

```diff
@@ -5389,17 +5389,72 @@ export class Skeleton {
     clone(): this;
 }
 
+/**
+ * A mesh that has a Skeleton with bones that can then be used to animate the vertices of the geometry. The material must support skinning and have skinning enabled.
+ * @see MeshStandardMaterial.skinning
+ * @source https://github.com/mrdoob/three.js/blob/master/src/objects/SkinnedMesh.js
+ */
 export class SkinnedMesh extends Mesh {
-    constructor(geometry?: Geometry | BufferGeometry, material?: Material | Material[], useVertexTexture?: boolean);
 
+    /**
+     * @param  {BufferGeometry} geometry? an instance of {@link BufferGeometry}.
```

Comment:
Please remove the types from the JSDoc in this PR. They can get out of date and Typescript doesn't use them for anything.

Reply:
Sorry, I'm a noob and I don't understand what you mean, can you specify what I have to remove from that line? I have checked three-core.d.ts and I see "@param" lines with the same format, so I'm not sure what's wrong.